open-metadata-resources/open-metadata-labs/ui-labs/ui-asset-search.ipynb

```diff
@@ -197,7 +199,7 @@
     "and we can login with the credential of Callie Quartile:\n",
     "```\n",
     "Username: calliequartile\n",
-    "Password: admin\n",
```

Comment:
could be worth also linking to the same page you have in the new lineage lab (line 110 in that notebook)? I think it's clearer? (this isn't a regression, just noticed when I compared against your better content!)

Reply:
already done (see lines 157:159 section copied from the other lab).